packages/marko-web/components/page/metadata/content.marko

```diff
@@ -1,12 +1,12 @@
 import gql from "graphql-tag";
 import { get } from "@parameter1/base-cms-object-path";
 import { warn } from "@parameter1/base-cms-utils";
-import buildStructuredData from "./google-structured-data/content";
+import defaultBuildStructuredData from "./google-structured-data/content";
 
 <!-- @todo This data should generated and saved to the content object as flat data, so no relationships are required. -->
 
 $ const { id } = input;
-$ const queryFragment = gql`
+$ const defaultQueryFragment = gql`
```

Comment:
Since the fragment defined here is required for this component to work, it would be be better to "spread" the custom fragment into the existing one.... this will prevent issues if the incoming fragment lacks any of the required fields needed by this component